api/tests/app/test_camera.py

```diff
@@ -0,0 +1,645 @@
+import pytest
+import os
+import base64
+import copy
+import numpy
+
+from api.tests.utils.common_functions import get_section_from_config_file, get_config_file_json
+
+# The line below is absolutely necessary. Fixtures are passed as arguments to test functions. That is why IDE could
+# not recognized them.
+from api.tests.utils.fixtures_tests import config_rollback, camera_sample, rollback_screenshot_camera_folder, h_inverse_matrix, pts_destination, rollback_homography_matrix_folder
+
+"""
+TO DO: Test stuffs related with the parameter reboot_processor.
```

Comment:
The format for a TODO is normally like this:

```python
# TODO: some message
```

This will be highlighted in some IDEs and is easier to search for.

BTW, just to make sure, is this TODO still valid?

Reply:
Thank you. And yes, it is still relevant.